www/creating-zooming-images.html

```diff
@@ -191,6 +196,31 @@ <h2>Creating Zooming Images</h2>
         <td>Free desktop app for locally viewing DZI files for Windows. Linux soon.</td>
         <td><a href="/examples/tilesource-dzi/">DZI</a></td>
     </tr>
+    
+    <tr>
+        <td colspan="3"><h4>Single-file Tile formats</h4> - see complete <a href="https://github.com/openseadragon/openseadragon/issues/944">discussion thread</a>. Useful for faster/easier file operations and maintenance on S3/file servers FS.</td>
+    </tr>
+    <tr>
+        <td>
+            szi_explorer <a href="https://github.com/infovore/szi_explorer">repo</a> and <a href="https://szi-explorer.netlify.app">demo</a>
+        </td>
+        <td>SZI is losslessly zipped DeepZoom container, and szi_explorer is the current OSD implementation by infovore and PolCPP. SZI can be produced with <a href="https://www.libvips.org/API/current/making-image-pyramids.html">libvips</a></td>
```

Comment:
What are your thoughts on using DZI here vs. Deep Zoom? I feel like on this page we use DZI more, so if we're trying to connect with that format, DZI would be more consistent. Is it literally a DZI container or is it somehow different so that just calling it Deep Zoom would be more accurate? If so, note that Deep Zoom should have a space between the words.